html/semantics/forms/the-fieldset-element/accessibility/shadow-dom-manual.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<title>fieldset accessibility test: shadow DOM</title>
+<template id="my-fieldset">
+  <fieldset>
+    <slot name="my-text"></slot>
+    <input>
+  </fieldset>
+</template>
+
+<my-fieldset>
+  <legend slot="my-text">Foo</legend>
+</my-fieldset>
+
+<p>Expected accessible name: ""
```

Comment:
mm, why?

Reply:
Accessible name computation works on the DOM tree, not the flat tree.